src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -128,6 +128,27 @@ private UserDetails buildUserDetails(Submission submission) {
 
     }
 
+    @Override
+    public ResponseEntity<GetPacakageInformationResponse> getPackageInformation(UUID id) {
+        Optional<Submission> fromCacheSubmission = this.submissionStore.getByKey(id);
+
+        if(!fromCacheSubmission.isPresent())
+            return ResponseEntity.notFound().build();
+
+        GetPacakageInformationResponse response = new GetPacakageInformationResponse();
```

Comment:
We should use mapStruct for building a `GetPacakageInformationResponse` from a `submission`